src/services/url.ts

```diff
@@ -92,19 +92,24 @@ class UrlService extends BaseService<UrlService.Options> {
     } else if (this.opts.redirects[url]) {
       WINDOW().location.assign(this.opts.redirects[url]);
     } else {
-      WINDOW().history.pushState({ url, state, app: STOREFRONT_APP_ID }, '', url);
+      WINDOW().history.pushState({ url, state: this.filterConfig(state), app: STOREFRONT_APP_ID }, '', url);
       this.app.flux.emit(Events.URL_UPDATED, url);
     }
   }
 
   replaceHistory(url: string) {
     WINDOW().history.replaceState({
       url,
-      state: this.app.flux.store.getState(),
+      state: this.filterConfig(this.app.flux.store.getState()),
       app: STOREFRONT_APP_ID
     }, WINDOW().document.title, url);
   }
 
+  filterConfig(state: Store.State) {
```

Comment:
I would rename this `filterState()` instead as we may want to use it to pull other elements out of the state in the future